src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -4283,4 +4296,29 @@ public String getJsonLd() {
         }
         return "";
     }
+
+    public void selectAllFiles() {
+        logger.info("selectAllFiles called");
+        List<FileMetadata> allFiles = new ArrayList<>();
+        for (FileMetadata fm : workingVersion.getFileMetadatas()) {
```

Comment:
why not just `selectedFiles = workingVersion.getFileMetadatas();`

Reply:
@oscardssmith good idea. Fixed in c592236.